metron-platform/metron-common/src/main/java/org/apache/metron/common/utils/HDFSUtils.java

```diff
@@ -29,6 +29,16 @@
 
 public class HDFSUtils {
 
+  public static byte[] readBytes(String path) throws IOException {
+    return readBytes(new Path(path));
+  }
+
+  public static byte[] readBytes(Path inPath) throws IOException {
+    FileSystem fs = FileSystem.get(inPath.toUri(), new Configuration());
+    FSDataInputStream inputStream = fs.open(inPath);
```

Comment:
I made this comment on @merrimanr 's PR, but it looks like it originated here.  This needs to be in a try-with-resources block otherwise we're leaking InputStreams.

Reply:
I found 2 such dangling input streams and am supplying a fix for both.